factory/factory.Dockerfile

```diff
@@ -62,10 +62,12 @@ ARG DEFAULT_NODE_VERSION
 ENV CYPRESS_DEFAULT_NODE_VERSION=${DEFAULT_NODE_VERSION}
 
 # Install Node: Node MUST be installed, so the default lives here
-ONBUILD ARG NODE_VERSION=${CYPRESS_DEFAULT_NODE_VERSION}
+ONBUILD ARG NODE_VERSION
+
+ONBUILD ENV NODE_VERSION_DEFAULT_APPLIED=${NODE_VERSION:-${CYPRESS_DEFAULT_NODE_VERSION}}
```

Comment:
i can only say node, version, and, default so many different ways 😓

Reply:
How about APPLIED_DEFAULT_NODE_VERSION? Is that accurate? Seems to maybe flow better.

Reply:
sure? i dunno!